static/src/javascripts/projects/common/modules/experiments/tests/global-eoy-header-test.js

```diff
@@ -124,5 +124,16 @@ export const globalEoyHeaderTest: ABTest = {
                 }
             },
         },
+        {
+            id: 'variant2',
+            test: (): void => {
+                const bar = getHeaderCtaBar();
+                if (bar) {
+                    const heading = month === 12 ? `Support us this December` : 'Support us in 2021';
```

Comment:
I guess we don't need this, but it doesn't really matter

Reply:
indeed, though this is probably all going to be deleted soon anyway!